bin/install_gems

```diff
@@ -1,16 +1,23 @@
 #!/bin/bash -eu
 
+dump_ruby_env_info "before running $(basename "$0")"
+
```

Comment:
I wonder if we still want to keep those debug dumps in the final `install_gems` action there?

Personally I'd remove both calls (before + after the `gem install bundler --version …`) to reduce verbosity — one can always temporarily add a call to `dump_ruby_env_info` in their pipeline directly if they need to debug Ruby issues on a given build later.

And if you still think it's useful to keep the dump of the whole Ruby env on every builds and steps calling `install_gem`, I'd still remove that initial one to keep only the one after we did the `gem install`

Reply:
My rationale for having them both is to see the before-after state. It was useful to me when trying to understand the issue, to see which was the selected Bundler as the pipeline started.

But given the fix in [114-gh-Automattic/buildkite-ci](https://href.li/?https://github.com/Automattic/buildkite-ci/issues/114) it's probably safe to remove the info logs, even though I don't think they made the logs particularly more verbose, thanks to Buildkite's logs grouping syntax–although I guess one could argue they _would_ make the raw logs more verbose.